### PR TITLE
changed wording for partition sys check

### PR DESCRIPTION
**Pull request by chaudum**
so it has the same tone as previous check messages

### CODE REVIEW

File: docs/sql/system.txt [hunk 1]
```diff
@@ -785,9 +785,9 @@ starts must not be less than default value of ``5m``.
 
 Number of partitions
 ....................
-This check produces a warning if any :ref:`partitioned table
-<partitioned_tables>` has more than 1000 partitions in order to detect the
-usage of a high cardinality field for paritioning.
+This check warns if any :ref:`partitioned table <partitioned_tables>` has more
+than 1000 partitions in order to detect the usage of a high cardinality field
+for paritioning.
```
Comment (mfelsche) [inline docs/sql/system.txt]: typo

**Comment by mfelsche** [comment]
the total number of acks for this PR must be greater than 0.

**Comment by mfelsche** [comment]
ack++

**Comment by chaudum** [comment]
https://github.com/crate/crate/commit/8e25bb2c40472b7f41c10157c9962d1a781e8e16

**Pull request closed without merging**